include/bitcoin/bitcoin/utility/random.hpp

```diff
@@ -19,6 +19,7 @@
 #ifndef LIBBITCOIN_RANDOM_HPP
 #define LIBBITCOIN_RANDOM_HPP
```

Comment:
Style, file names should change to `pseudo_random` to match class (and the include guards above).